examples/run_examples.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+#
+# examples/run_examples.py -- run some examples.
+#
+#	Copyright (C) 2008 David Bronke <whitelynx@gmail.com>
+#	Copyright (C) 2002 Peter Liljenberg <petli@ctrl-c.liu.se>
```

Comment:
Copyrights must be updated. :)
Please also use LGPL v2.1 license to every new file. The contributors agreed on changing the license to LGPL v2.1.
Example of the LGPL license caption can be found here: https://github.com/pywinauto/pywinauto/blob/master/pywinauto/SendKeysCtypes.py